### PR TITLE
[wip] ui: link to debug pages in sidebar

### LINKED ISSUE #20690: ui: surface debug pages for users
cc @josueeee your requested user stories for the debug page in 2.0

- I want to know how my nodes are configured (node diagnostics page) - reports/nodes
- I want to know which ranges are causing problems (problem ranges report) - /reports/problemranges
- I want to know if my nodes can talk to each other on the network (network diagnostics page) - reports/network
- I want to check if my certificates are configured properly (certificates page) - reports/certificates/local
- [New] I want to see what my localities are (locality tree)
- [New] I want to check my zone configs (replica matrix)

Usability
- Flag this feature as "experimental / beta"
- Segment debug options into standard and advanced
- Link to user contract on what it means for a feature to be experimental

and cc @BramGruneir for thoughts. For context, we don't have a lot of time to brush the debug pages up, so this is incredibly limited in scope. Basically trying to highlight what would be most useful to users who are not CockroachDB experts.

### PR DESCRIPTION
**Pull request by vilterp**
image

image

TODO:
- [x] some more copy on the debug page index with a disclaimer about their status and what some of the debug pages do (copy is in the issue)
- [x] add descriptions to most useful debug pages
- [ ] pull most useful ones to the top & put under a new heading
- [ ] the icon seems to be getting cut off a little bit on the sides
- [x] fix stuff in logout icon being underlined, now that there's a link around it

Fixes #20690

### CODE REVIEW

File: pkg/ui/src/views/app/components/layoutSidebar/index.tsx [hunk 1]
```diff
@@ -88,13 +89,17 @@ function LoginIndicator({ loginState, handleLogout }: LoginIndicatorProps) {
...
-        <div
-          className="login-indicator__initial"
-          title={`Signed in as ${user}`}
-        >
-          { user[0] }
+      <Link to={LOGOUT_PAGE} onClick={handleLogout}>
+        <div>
+          <div
+            className="login-indicator__initial"
+            title={`Signed in as ${user}`}
```
Comment (couchand) [inline pkg/ui/src/views/app/components/layoutSidebar/index.tsx]: as long as you're in here, can you update this to use `TooltipWrapper` please?

File: pkg/ui/src/views/app/components/layoutSidebar/index.tsx [hunk 2]
```diff
@@ -128,6 +137,13 @@ export default class Sidebar extends React.Component {
           <IconLink to="/jobs" icon={jobsIcon} title="Jobs" />
         </ul>
         <ul className="navigation-bar__list navigation-bar__list--bottom">
+          <li>
+            <Link to="/debug" className={classNames("debug-pages-link", { active: this.context.router.isActive("/debug") })}>
```
Comment (couchand) [inline pkg/ui/src/views/app/components/layoutSidebar/index.tsx]: please use `IconLink`

Comment (vilterp) [reply pkg/ui/src/views/app/components/layoutSidebar/index.tsx]: Tried to use `IconLink` — it pulls in a lot of CSS which I was unable to finagle to get the spacing I wanted. Maybe will be more successful today.

**Comment by couchand** [comment]
also closes #29310

File: pkg/ui/src/views/app/components/layoutSidebar/index.tsx (hunk 1 above)
Comment (vilterp) [reply pkg/ui/src/views/app/components/layoutSidebar/index.tsx]: More CSS struggles when I insert the `ToolTipWrapper` :P 
image

**Comment by Amruta-Ranade** [comment]
Edits for note at the top of the page:
The following pages are meant for advanced monitoring and troubleshooting. Note that the pages are experimental and undocumented. If you find an issue, let us know through these channels .

**Comment by couchand** [comment]
closing in favor of  #31218

**Pull request closed without merging**